website/docs/tutorials/04-configuration-file.md

```diff
@@ -35,65 +38,98 @@ network = "testnet"
+The `[contracts]` section allows you to define contract names and the corresponding files that make up each contract.
+This is useful because it allows you to refer to contracts by name rather than having to specify the full file path each time.
+You can also combine multiple files into a single contract.
+Currently, the `[contracts]` section is required by the [`protostar build`](/docs/cli-reference#build) command.
+
+Here is an example of how to define a contract in the [contracts] section:
+
 ```toml
 [contracts]
 my_contract = ["src/feature_a.cairo", "src/feature_b.cairo"]   
 ```
 
-:::note
-Currently, [`protostar build`](/docs/cli-reference#build) requires this section.
-:::
-
 ### `[COMMAND]`
-Configure command arguments, to avoid passing them manually. CLI can be represented in the configuration file in the following way:
+The `[COMMAND]` section of the `protostar.toml` file allows you to specify options and settings for a specific Protostar command.
 
-```console title="CLI"
-protostar format src tests --ignore-broken
+To configure command arguments in the `protostar.toml` file, you can specify them in a `[COMMAND]` section using the following format:
+
+```toml
+[COMMAND]
+flag = true
+list = [1, 2, 3]
+path = "./src"
 ```
+
+For example, the following configuration file specifies the [`target`](/docs/cli-reference#target-string) and [`ignore-broken`](/docs/cli-reference#--ignore-broken) arguments for the [`protostar format` command](/docs/cli-reference#format):
+
 ```toml title="Configuration File"
 [format]
 target = ["src", "tests"]
 ignore-broken = true
 ```
 
-Check the [CLI Reference](/docs/cli-reference) or run `protostar COMMAND --help` to learn supported arguments.
+You can then run the [`protostar format`](/docs/cli-reference#format) command without specifying the [`target`](/docs/cli-reference#target-string) and [`ignore-broken`](/docs/cli-reference#--ignore-broken) arguments in the console:
+
+```console title="CLI"
+protostar format
+```
+
+To learn more about the available options and arguments for each Protostar command, refer to the [CLI Reference](/docs/cli-reference) or run `protostar COMMAND --help` to see the list of supported arguments.
 
 ### Configuration Profiles
-Configuration profiles provide a way to easily switch between Protostar configurations. Profiles inherit values from non-profiled configuration. In order to create a configuration profile, add a new section in protostar.toml with the following naming convention:
--  `[profile.PROFILE_NAME.project]` - to create a profile for a [Shared Configuration](#shared-configuration)
--  `[profile.PROFILE_NAME.COMMAND]` - to create a profile for a [Command Configuration](#command)
+Configuration profiles allow you to easily switch between different Protostar configurations. 
+When you activate a profile, it will override the default settings specified in the `protostar.toml` file with the settings specified in the profile.
```

Comment:
We don't really "activate" it per-se though, do we?

Reply:
I am not sure if I get it. User activates a profile. "activate" seems a right word to me.

> activate — make (something) active or operative.
> operative — functioning or having effect

Reply:
We really activate + de-activate it (since it turns off after the command finishes), so i'd look for another word here. I'd probably go for something like "passing a profile flag will override existing configuration" since that's what's really happening from the user perspective.